phobos_umss.sh

```diff
@@ -0,0 +1,490 @@
+        #     "device"  - device ID of device containing file
+        #     "inode"   - file serial number
+        #     "mode"    - ACL mode in octal
+        #     "nlink"   - number of hard links to the file
+        #     "uid"     - user id of file
+        #     "gid"     - group id of file
+        #     "devid"   - device id
+        #     "size"    - file size
+        #     "blksize" - block size in bytes
+        #     "blkcnt"  - number of blocks
+        #     "atime"   - last access time
+        #     "mtime"   - last modification time
+        #     "ctime"   - last change time
+        #
+        # e.g: device=$(echo $output | awk '{print $3}')
+        # Note 1: if some of these parameters are not relevant, set them to 0.
+        # Note 2: the time should have this format: YYYY-MM-dd-hh.mm.ss with:
+        #                                           YYYY = 1900 to 2xxxx, MM = 1 to 12, dd = 1 to 31,
+        #                                           hh = 0 to 24, mm = 0 to 59, ss = 0 to 59
+        md_str=""
+        md_str+="device=$( echo "$output" | sed -nr 's/.*\<Device: *(\S*)\>.*/\1/p'),"
+        md_str+="inode=$(  echo "$output" | sed -nr 's/.*\<Inode: *(\S*)\>.*/\1/p'),"
+        md_str+="mode=$(   echo "$output" | sed -nr 's/.*\<Access: *\(([0-9]*)\/.*/\1/p'),"
+        md_str+="nlink=$(  echo "$output" | sed -nr 's/.*\<Links: *([0-9]*)\>.*/\1/p'),"
+        md_str+="uid=$(    echo "$output" | sed -nr 's/.*\<Uid: *\( *([0-9]*)\/.*/\1/p'),"
+        md_str+="gid=$(    echo "$output" | sed -nr 's/.*\<Gid: *\( *([0-9]*)\/.*/\1/p'),"
+        md_str+="devid=0,"
+        md_str+="size=$(   echo "$output" | sed -nr 's/.*\<Size: *([0-9]*)\>.*/\1/p'),"
+        md_str+="blksize=$(echo "$output" | sed -nr 's/.*\<IO Block: *([0-9]*)\>.*/\1/p'),"
+        md_str+="blkcnt=$( echo "$output" | sed -nr 's/.*\<Blocks: *([0-9]*)\>.*/\1/p'),"
+        md_str+="atime=$(  echo "$output" | sed -nr 's/.*\<Access: *([0-9]{4,}-[01][0-9]-[0-3][0-9]) *([0-2][0-9]):([0-5][0-9]):([0-6][0-9])\..*/\1-\2.\3.\4/p'),"
+        md_str+="mtime=$(  echo "$output" | sed -nr 's/.*\<Modify: *([0-9]{4,}-[01][0-9]-[0-3][0-9]) *([0-2][0-9]):([0-5][0-9]):([0-6][0-9])\..*/\1-\2.\3.\4/p'),"
+        md_str+="ctime=$(  echo "$output" | sed -nr 's/.*\<Change: *([0-9]{4,}-[01][0-9]-[0-3][0-9]) *([0-2][0-9]):([0-5][0-9]):([0-6][0-9])\..*/\1-\2.\3.\4/p')"
+        echo "${md_str}"
+    fi
+
+    save_log get_metadata "END get_metadata($*)"
+    return $error_9
+}
+
+
+
+save_log ""
+save_log "##########################"
+save_log "# BEGIN EXPERIMENT $ID #"
+save_log "##########################"
+if [ $DISPLAY_COLOUR == 1 ]; then save_log "$Blue$0_$Purple$VERSION"; fi
+if [ $DISPLAY_BLACK == 1 ]; then save_log "$0_$VERSION"; fi
+
+#get_daemon=$(get_phobosd)
+#if [ $? != 0 ]; then exit $?; fi
+#check_avail_dir=$(check_dir)
+#if [ $? != 0 ]; then exit $?; fi
+
+
+#############################################
+# below this line, nothing should be changed.
```

Comment:
question: why? :D.

Reply:
Because this comment was already in the template file of iRODS UnivMSS.
Under this line, the script calls the appropriate function given as its first argument and forwards its arguments.
You can modify things after this line, but at your own risks...